### PR TITLE
Document how AppCat will adapt odoo 16 billing

### PR DESCRIPTION
**Pull request by Kidswiss**
## Summary
Documentation how Schedar plans to adapt the new billing.

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues if applicable.

### CODE REVIEW

**Review by wejdross — APPROVED** [review]
I went throught links and Your idea and I think it's ok. It's quite high level so far, but looks correctly for me

File: docs/modules/ROOT/pages/reference/cloud-usage-reporting.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+This configuration is abridged from the https://github.com/vshn/billing-collector-cloudservices/blob/master/component/class/defaults.yml[existing config] and only contains example configs for the new Odoo 16 billing.
+
+[code:yaml]
+----
+parameters:
+  billing_collector_cloudservices:
+    billingEndpoint:
+      URL: https://odoo16.vshn.ch <1>
+      API_TOKEN: topsecret <2>
```
Comment (TheBigLee) [inline docs/modules/ROOT/pages/reference/cloud-usage-reporting.adoc:27]: On APPUiO Managed clusters, that would expose the token to write to odoo16 to the customers and they can potentially directly interact with odoo and write arbitrary data in it (or in the worst case, delete data)

Comment (zugao) [reply docs/modules/ROOT/pages/reference/cloud-usage-reporting.adoc:27]: Yes this is a security concern that must be raised with Aldebaran. I'm pretty sure though the data cannot be deleted or updated once you send it into the odoo16 queue.

Comment (Kidswiss) [reply docs/modules/ROOT/pages/reference/cloud-usage-reporting.adoc:27]: @TheBigLee asked tobru about this. He is looking into it.

**Review by zugao — APPROVED** [review]
LGTM

**Pull request merged**